website/docs/index.html.markdown

```diff
@@ -179,13 +179,20 @@ In addition to [generic `provider` arguments](https://www.terraform.io/docs/conf
 
 For more information on configuring and managing programmatic API Keys see the [MongoDB Atlas Documentation](https://docs.atlas.mongodb.com/tutorial/manage-programmatic-access/index.html).
 
-## HashiCorp Terraform Version Requirement
-| MongoDB Atlas Provider version  | Required Terraform version |
-| ------------- | ------------- |
-| `1.12.0` and above  | `1.0+`  |
-| below `1.12.0`  | `0.13+` |
-
-* While we support minimum of HashiCorp Terraform version 1.0+ (as we have migrated to [Plugin Protocol Version 6](https://developer.hashicorp.com/terraform/plugin/terraform-plugin-protocol#protocol-version-6)), for your safety we strongly recommend only consuming versions of Terraform that are currently receiving Security/Maintenance Updates. For latest Terraform versions see: https://endoflife.date/terraform 
+## [HashiCorp Terraform Version](https://www.terraform.io/downloads.html) Compatibility Matrix
```

Comment:
[q] do we plan to include a mention in our next release guide about the Compatibility matrix to notify users? Or would the changelog suffice?

Reply:
Suggest we mention in both places with link to Terraform Docs. Added additional commit with updates to upcoming Upgrade Guide. ChangeLog will be pulled in automatically from the ChangeLog auto-generation, will also manually adjust as needed when we get there.